src/foam/lib/formatter/JSONFObjectFormatter.java

```diff
@@ -184,13 +184,13 @@ protected void outputProperty(FObject o, PropertyInfo p) {
     p.formatJSON(this, o);
   }
 
-  protected boolean maybeOutPutFObjectProperty(FObject newFObject, FObject oldFObject, PropertyInfo prop) {
+  protected boolean maybeOutputFObjectProperty(FObject newFObject, FObject oldFObject, PropertyInfo prop) {
```

Comment:
these  **maybeOutPut** casing specific names don't seem  to  be found anywhere else  so should  be  good to change